incubator/java-openliberty/image/project/Dockerfile

```diff
@@ -18,7 +18,7 @@ COPY ./validate.sh /project/user-app/validate.sh
 RUN cd /project/user-app && ./validate.sh
 
 # Copy and build the application source, and run unit tests
-COPY ./user-app/src /project/user-app/src
+COPY ./user-app /project/user-app
 RUN cd /project/user-app && mvn liberty:create package
 
 RUN cd /project/user-app/target/liberty/wlp/usr/servers && \
```

Comment:
I think we should go further and separately copy the libs and resources, e.g. something like:
```
 # Stage 2: Package Open Liberty image
 FROM open-liberty:kernel-java8-openj9

 # 2a) shared resources change the least-frequently  (* - see note below)
 COPY --chown=1001:0 --from=0 /shared/resources /opt/ol/wlp/usr/shared/

 # 2b) next copy shared library
 COPY --chown=1001:0 --from=0 /config/libs/ /config/

 # 2c) Server config, bootstrap.properties, and everything else
 COPY --chown=1001:0 --from=0 /config/ /config/

 # 2d) Changes to the application binary
 COPY --chown=1001:0 --from=0 /project/user-app/target/*.war /config/apps
```

Reply:
done